src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -145,7 +146,7 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
      */
     @Advanced
     @Argument(fullName = "input_prior", shortName = "inputPrior", doc = "Input prior for calls", optional = true)
-    public List<Double> inputPrior = Collections.emptyList();
+    public List<Double> inputPrior = new ArrayList<>();
```

Comment:
I think it was a serialization issue, not a CLI issue, I can't remember though and I don't see any failures in my tests when I switch it back.